packages/core/src/secretStore/secretKeyGenerator.test.ts

```diff
@@ -17,135 +17,50 @@
  *
  */
 
-import {Decoder, Encoder} from 'bazinga64';
+import {EncryptedStore, createCustomEncryptedStore, createEncryptedStore} from './encryptedStore';
+import {generateSecretKey} from './secretKeyGenerator';
 
-import {generateSecretKey, CorruptedKeyError} from './secretKeyGenerator';
-
-const systemCryptos = {
-  v0: {
```

Comment:
correct, this logic now completely lives in the webapp. since the webapp version and the core version are always corrolated, we will not need to handle backward compat (see https://github.com/wireapp/wire-webapp/pull/16491/files#diff-5d5af779079c6b10bb91ed68e976c0bad494d65f6fdca4b1c4c30a85707363ecR1)
But that is an excellent question :)